README.md

```diff
@@ -32,7 +34,7 @@ Load data and scripts onto Yeti (yeti.cr.usgs.gov, AD login). Include all 6 mode
 
 1. Login to Yeti, `ssh user@yeti.cr.usgs.gov`
 1. If you are a windows user, you may need to run `dos2unix split_historic_data.slurm` before continuing to make the line endings correct for each of the `.slurm` files before using them.
-1. Enter the R environment and install needed R packages for all the R scripts. You will need: ncdf4, dplyr, purrr and tidyr. Type `R` at the command prompt and then `install.packages("ncdf4","dplyr","purrr","tidyr")` and <enter> once you're in the environment. When it is installing it'll ask if you want to create a personal libraries to put these packages, type `yes` to create a personal library, pick a CRAN mirror, and ensure that the package installs. To quit R, type `q()`, and choose whether or not to save the session.
+1. Enter the R environment and install needed R packages for all the R scripts. You will need: ncdf4, dplyr, purrr and tidyr. Type `R` at the command prompt and then `install.packages("ncdf4","dplyr","purrr","tidyr","pryr","data.table")` and <enter> once you're in the environment. When it is installing it'll ask if you want to create a personal libraries to put these packages, type `yes` to create a personal library, pick a CRAN mirror, and ensure that the package installs. To quit R, type `q()`, and choose whether or not to save the session.
```

Comment:
The packages arg in `install.packages` takes a vector, e.g. `install.packages(c("ncdf4", "RNetCDF")`